include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -376,7 +377,7 @@ struct OrtApi {
   OrtStatus*(ORT_API_CALL* SessionGetOverridableInitializerTypeInfo)(_In_ const OrtSession* sess, size_t index, _Outptr_ OrtTypeInfo** type_info)NO_EXCEPTION;
 
   /**
-   * \param value  is set to a null terminated string allocated using 'allocator'. The caller is responsible in freeing it.
+   * \param value  is set to a null terminated string allocated using 'allocator'. The caller is responsible to free it.
```

Comment:
>The caller is responsible to free it [](start = 83, length = 36)

nit: 'responsible for freeing it' or 'required to free it'

Reply:
Fixed.